### ISSUE TITLE
Is it possible to lint code comments and/or docblocks?

### ISSUE THREAD

**Issue by jgornick** [get-alex/alex#322]
As the title says :)

I suppose one workaround for linting docblocks is linting any generated documentation from the docblocks.

Thank you!

**Comment by wooorm** [comment]
nope, there’s not really support for that. Although, you can pass `--text` on the CLI to see the files you give it as plain text.
Alternatively, you could do the inverse based on what linter exists for your code, and integrate that to look for code comments, and run the alex API on those comments!

**Comment by boomshadow** [comment]
I use a combination of [`fd`](https://github.com/sharkdp/fd) (a `find` replacement) and what @wooorm suggested above to get Alex as a linter in my projects' CI/CD pipeline. I find all ruby & markdown files, then pass to Alex to be linted as text:

```bash
fd -e rb -e md -x alex --text {}
```
